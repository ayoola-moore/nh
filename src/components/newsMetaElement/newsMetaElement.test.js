import React from "react";
import NewsMetaElement from "./";

describe("Component", () => {
  const component = shallow(<NewsMetaElement />);
  it("should render NewsMetaElement properly", () => {
    expect(component).toMatchSnapshot();
  });
});
