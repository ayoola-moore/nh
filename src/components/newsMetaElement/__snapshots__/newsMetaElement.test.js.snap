// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render NewsMetaElement properly 1`] = `ShallowWrapper {}`;
